test/parallel/test-no-enter-tickcallback.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+var allsGood = false;
+var cntr = 0;
+
+process.on('exit', () => {
+  assert.ok(cntr > 0, '_tickDomainCallback was never called');
+});
```

Comment:
@addaleax Though on my machine it would be `common.mustCall(., 3)`. Because we can't actually process the `nextTickQueue` when overriding `_tickDomainCallback()` we can only guarantee it will be called at least once. I don't feel that being any stricter than that is possible w/o making some shaky assumptions on how the internals work.